jbmc/unit/Makefile

```diff
@@ -126,16 +123,19 @@ OBJ += $(CPROVER_LIBS) \
        $(CPROVER_DIR)/unit/testing-utils/testing-utils$(LIBEXT) \
        java-testing-utils/java-testing-utils$(LIBEXT)
 
-TESTS = unit_tests$(EXEEXT) \
-        # Empty last line
+CATCH_TEST = unit_tests$(EXEEXT)
+N_CATCH_TESTS = 84
```

Comment:
Bit of a pain, but can't immediately think of a better way. But this should probably be documented in the Catch documentation.

Reply:
One idea, but don't know how fiddly to implement - count `SCENARIO` + `TEST_CASE` in all `.cpp` files in `unit`

```bash
$ cat unit/*.cpp | grep -c -E "(SCENARIO|TEST_CASE)"
```
Which seems to work but I only get 26 so must be doing something wrong

Reply:
It was the `**` that wouldn't quite expand the way you'd hope it would do :-) - I have implemented a variant thereof using `find`.